apps/dashboard/config/configuration_singleton.rb

```diff
@@ -189,15 +190,6 @@ def xdmod_integration_enabled?
     xdmod_host.present?
   end
 
-  # Support ticket configuration
-  def support_ticket_config
-    config.fetch(:support_ticket, {})
-  end
-
-  def support_ticket_enabled?
-    !support_ticket_config.empty?
-  end
-
```

Comment:
I'd say we keep this. I like the implicit enabling through configuration. We can still read through configs and search for it.  

```
cfg[:profiles].any? { |_, p| p.has_key?(:support_ticket) }
```

Yes there's a little knowledge sharing, but we're weighing that against the extra/redundant configuration.